libbeat/kibana/client.go

```diff
@@ -271,7 +274,7 @@ func (client *Client) ImportJSON(url string, params url.Values, jsonBody map[str
 
 	body, err := json.Marshal(jsonBody)
 	if err != nil {
-		logp.Err("Failed to json encode body (%v): %#v", err, jsonBody)
+		client.log.Errorf("Failed to json encode body (%v): %#v", err, jsonBody)
```

Comment:
I wonder if this log is redundant. The error returned also contains the original error message. The jsonBody maybe we always want to log at debug level?

Would it make sense to add debug logs to other places? E.g. before and after a request is send?

Reply:
I've just run `setup --dashboards` with debug logging. It prints which dashboard it is loading and if there is an error it is logged, too. I think that should be enough.